Test Failure: MPConcurrentTest.testCompletionStageFactorySupplyAsync

Test Failure: com.ibm.ws.concurrent.mp.fat.MPConcurrentTest.testCompletionStageFactorySupplyAsync

```
testCompletionStageFactorySupplyAsync:junit.framework.AssertionFailedError: 2020-09-09-10:55:04:606 The response did not contain '[SUCCESS]'.  Full output is:'
ERROR: Caught exception attempting to call test method testCompletionStageFactorySupplyAsync on servlet concurrent.mp.fat.web.MPConcurrentTestServlet
java.lang.AssertionError: ForkJoinPool.commonPool-worker-0 expected:<-1> but was:<0>
at concurrent.mp.fat.web.MPConcurrentTestServlet.testCompletionStageFactorySupplyAsync(MPConcurrentTestServlet.java:1652)
at componenttest.app.FATServlet.doGet(FATServlet.java:71)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:686)
at javax.servlet.http.HttpServlet.service(HttpServlet.java:791)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1230)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:729)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.handleRequest(ServletWrapper.java:426)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1226)
at com.ibm.ws.webcontainer.filter.WebAppFilterManager.invokeFilters(WebAppFilterManager.java:1010)
at com.ibm.ws.webcontainer.servlet.CacheServletWrapper.handleRequest(CacheServletWrapper.java:75)
at com.ibm.ws.webcontainer40.servlet.CacheServletWrapper40.handleRequest(CacheServletWrapper40.java:83)
at com.ibm.ws.webcontainer.WebContainer.handleRequest(WebContainer.java:936)
at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost$2.run(DynamicVirtualHost.java:279)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink$TaskWrapper.run(HttpDispatcherLink.java:1134)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.wrapHandlerAndExecute(HttpDispatcherLink.java:415)
at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.ready(HttpDispatcherLink.java:374)
at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleDiscrimination(HttpInboundLink.java:577)
at com.ibm.ws.http.channel.internal.inbound.HttpInbound
```

The test case assumes that Executors.newFixedThreadPool will never be a ForkJoin pool, but there is nothing in the JavaDoc that guarantees this.  The assert on thread name needs to be removed from the test.

--- a/dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentApp/src/concurrent/mp/fat/web/MPConcurrentTestServlet.java
+++ b/dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentApp/src/concurrent/mp/fat/web/MPConcurrentTestServlet.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017,2019 IBM Corporation and others.
+ * Copyright (c) 2017,2020 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
@@ -1647,16 +1647,14 @@ public class MPConcurrentTestServlet extends FATServlet {
         CompletableFuture<Object> cf2 = cf1.thenApplyAsync(lookUpResourceRef);
         CompletableFuture<String> cf3 = cf2.thenApplyAsync(getThreadName);
 
-        threadName = cf3.get(TIMEOUT_NS, TimeUnit.NANOSECONDS);
+        threadName = cf1.get(TIMEOUT_NS, TimeUnit.NANOSECONDS);
         assertFalse(threadName, threadName.startsWith("Default Executor-thread-")); // not on Liberty global thread pool
-        assertEquals(threadName, -1, threadName.toUpperCase().indexOf("FORK")); // not on fork join pool
 
         lookupResult = cf2.get(TIMEOUT_NS, TimeUnit.NANOSECONDS);
         assertTrue(lookupResult.toString(), lookupResult instanceof NamingException); // no access to application's namespace
 
         threadName = cf3.get(TIMEOUT_NS, TimeUnit.NANOSECONDS);
         assertFalse(threadName, threadName.startsWith("Default Executor-thread-")); // not on Liberty global thread pool
-        assertEquals(threadName, -1, threadName.toUpperCase().indexOf("FORK")); // not on fork join pool
 
         // there should not be any context clearing without a managed executor service
         lookupResult = cf3.thenApply(lookUpResourceRef).get(TIMEOUT_NS, TimeUnit.NANOSECONDS);

